plugins/search-react/src/components/SearchFilter/SearchFilter.stories.tsx

```diff
@@ -64,6 +64,18 @@ export const SelectFilter = () => {
   );
 };
 
+export const MultiSelectFilter = () => {
```

Comment:
Could you add examples with placeholder, renderValue and emptyItemLabel?

Reply:
done